pkg/frontend/adminactions/kubeactions.go

```diff
@@ -91,11 +92,18 @@ func (k *kubeActions) KubeList(ctx context.Context, groupKind, namespace string)
 		return nil, err
 	}
 
-	ul, err := k.dyn.Resource(*gvr).Namespace(namespace).List(ctx, metav1.ListOptions{})
+	// protect RP memory by not reading in more than 1000 items
```

Comment:
I don't love this, but I think it's pragmatic for now.

Reply:
The JMESPath query runs after the data is fetched correct?  

I think we have some queries that check for pods in crashloopbackoff or without limits which this might inhibit until we get over to JIT auto approval on the portal.

Reply:
Sorry for naive question, can those const values like the number of records per page be part of a conf file so that we can somehow override/tweak them and avoid a full delivery of the RP ?